pkg/api/validation/validation.go

```diff
@@ -3189,6 +3178,9 @@ func ValidateSecurityContext(sc *api.SecurityContext, fldPath *field.Path) field
 			allErrs = append(allErrs, field.Invalid(fldPath.Child("runAsUser"), *sc.RunAsUser, isNegativeErrorMsg))
 		}
 	}
+
+	// TODO when the versioned object is using feilds change this field path
+	allErrs = append(allErrs, ValidateSeccompProfile(sc.SeccompProfile, field.NewPath("pod", "metadata", "annotations", api.SeccompContainerAnnotationKeyPrefix+containerName))...)
```

Comment:
Hrm. Yeah, it's going to be wrong for at least one case either way we go. 

I think what I'm going to do is separate the field move out from this PR just to reduce the scope for reviews of that portion since it causes changes in more than just psp.